### DISCUSSION TITLE
We really miss Model's Stage transition date

### DISCUSSION THREAD

**Discussion by smolendawid** [mlflow/mlflow#6033] [Ideas]
Transitioning an MLflow Model’s Stage is extremely useful in my opinion but the information we really miss there is when the transition happens. Versioning itself is helpful but the real power comes from being able to track the versions in the whole deployment lifecycle. 

For example, for now, it's impossible to show which version performed how well for what period of time, as we only store the stage for a given Version, no metadata for transitions. In practice it's a limitation that prevents you from comparing the Versions performance after they are promoted to Production and work on real data.

**Comment by BenWilson2** [comment]
Hi @smolendawid thank you for the suggestion on how to build more functionality into the model registry service. Could I trouble you to file a FR issue for this wherein you detail in full exactly the functionality that you'd like to see? 
I think this has serious merit and want to make sure that we have the full image in mind of what you're looking for the service to do so that we can discuss the feasibility of getting a design together and potentially put on the roadmap. 
Thank you for the feedback and I hope that you're able to file that FR!
